RecoVertex/PrimaryVertexProducer/src/DAClusterizerInZ_vect.cc

```diff
@@ -662,6 +662,9 @@ bool DAClusterizerInZ_vect::split(const double beta, track_t& tks, vertex_t& y,
 
   std::vector<std::pair<double, unsigned int>> critical;
   for (unsigned int k = 0; k < nv; k++) {
+    if (y.sw[k] < 1.0e-100) {
+      continue;
+    }
     double Tc = 2 * y.swE[k] / y.sw[k];
```

Comment:
`swE` is apparently of a similar order as `sw`, although the more precise relationship is complicated. I think that the result should be skipped only if `not isFinite(Tc)` .

Reply:
Something like:

```cpp
if (std::isfinite(Tc) and beta * Tc > threshold) {
      critical.push_back(make_pair(Tc, k));
}
```

?